LoopKit/DosingDecisionStore.swift

```diff
@@ -227,6 +230,33 @@ public struct StoredDosingDecision {
             self.date = date
         }
     }
+
+    public struct DeviceSettings: Codable, Equatable {
+        let name: String
+        let systemName: String
+        let systemVersion: String
+        let model: String
+        let modelIdentifier: String
+        let batteryLevel: Float?
```

Comment:
no serial number?  I just thought it might be useful (though, maybe not every device can provide it...?)

Reply:
AFAIK, no phone reports its serial number to non-Apple apps. This would be too easy to track.

Reply:
Oh sheesh, my bad, I thought this was Settings for an _external_ device (eg. CGM, Pump).  Could be disambiguated by naming this `IOSDeviceSettings`...